app/notification.go

```diff
@@ -284,7 +284,7 @@ func (a *App) SendNotifications(post *model.Post, team *model.Team, channel *mod
 	if *a.Config().EmailSettings.SendPushNotifications {
 		pushServer := *a.Config().EmailSettings.PushNotificationServer
 		if license := a.License(); pushServer == model.MHPNS && (license == nil || !*license.Features.MHPNS) {
-			mlog.Warn("api.post.send_notifications_and_forget.push_notification.mhpnsWarn FIXME: NOT FOUND IN TRANSLATIONS FILE")
+			mlog.Warn("Failed to get the license or check the MHPNS feature. Push notification is disabled")
```

Comment:
Is it that Mattermost cannot
 - get the EE license
 - confirm if the Mattermost Hosted Push Notification Service is enabled

and hence push notifications have been disabled?

Reply:
MM cannot get the license or the license does not have the NHPNS feature

Reply:
By `doesn't have`, is that possible? Just wondering why an admin might hit that error, or if it's someone who would have forked Mattermost and removed the feature?

Reply:
@cpanato Figuring out what would be the appropriate text. Push notifications can be enabled without an EE license, and I'm not sure why someone wouldn't have the HPNS service available..

Perhaps

```
Push notifications are disabled. Go to System Console > Notifications > Mobile Push to enable them.
```

?

Reply:
sounds good this message